registry-api/src/main/scala/au/csiro/data61/magda/registry/Protocols.scala

```diff
@@ -1,17 +1,15 @@
 package au.csiro.data61.magda.registry
 
-import spray.json.{DefaultJsonProtocol, JsNumber, JsValue, RootJsonFormat}
+import spray.json._
+import au.csiro.data61.magda.model.Registry._
 import gnieh.diffson.sprayJson._
+import au.csiro.data61.magda.model.Registry.{ Protocols => CommonRegistryProtocols }
+import au.csiro.data61.magda.registry.DeleteResult
+import au.csiro.data61.magda.registry.BadRequest
 
-trait Protocols extends DefaultJsonProtocol with DiffsonProtocol with au.csiro.data61.magda.model.temporal.Protocols {
-  implicit object EventTypeFormat extends RootJsonFormat[EventType] {
-    def write(e: EventType) = JsNumber(e.value)
-    def read(value: JsValue) = EventType.withValue(value.asInstanceOf[JsNumber].value.toInt)
-  }
+trait Protocols extends DefaultJsonProtocol with DiffsonProtocol with au.csiro.data61.magda.model.Temporal.Protocols with CommonRegistryProtocols {
 
   implicit val badRequestFormat = jsonFormat1(BadRequest.apply)
-  implicit val aspectFormat = jsonFormat3(AspectDefinition.apply)
-  implicit val recordFormat = jsonFormat3(Record.apply)
   implicit val recordSummaryFormat = jsonFormat3(RecordSummary.apply)
```

Comment:
Should we move some more of these into Common?  Even if they're not needed there yet, it seems a bit odd not to have `BadRequest` and `RecordSummary` given the others that have been moved there.